tests/test-dt-stats.py

```diff
@@ -235,7 +235,37 @@ def test_dt_sd_special_cases(src, res):
     assert list_equals(dtr.to_list(), [[res]])
 
 
+#-------------------------------------------------------------------------------
+# Skew function dt.skew()
+#-------------------------------------------------------------------------------
+
+def truncate(n, decimals=0):
+    multiplier = 10 ** decimals
+    return int(n * multiplier) / multiplier
```

Comment:
You can just use the Python's `math.isclose()` instead.

Reply:
Okay

Reply:
by the way, when using `math.isclose()`, what do you think should be the appropriate `abs_tol`?

Reply:
I normally start with the default ones, i.e. `rel_tol=1e-09, abs_tol=0.0`, and slightly adjust them if needed. If the adjustment required is more than several orders of magnitude, this is normally a sign to check the source of the discrepancy.

Reply:
It's failing for `rel_tol < 1e-05` and `abs_tol < 1e-04`, which is many orders of magnitude higher than the default values. To be honest, I only worked on providing user-level API's and assumed care has been taken with the implementation. Now that the discrepancy is this high, I feel compelled to look deeply into the implementation, which I can do but this may not be ready for the upcoming release. What do you think?

Reply:
In this case, since we don't know the `abs_tol` in advance, it is reasonable to use `rel_tol` and see how far away we're from, let's say, pandas.